specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -2156,6 +2195,72 @@
         "password"
       ]
     },
+    "SSISAccessCredential": {
```

Comment:
>SSISAccessCredential [](start = 5, length = 20)

This must be excluded in GET calls.. to avoid revealing secrets.

Reply:
so the actual resource requires different schema for GET

---
In reply to: [300815583](https://github.com/Azure/azure-rest-api-specs/pull/6548#discussion_r300815583) [](ancestors = 300815583)

Reply:
We don't have GET calls on the resource.